src/SFML/Window/OSX/HIDInputManager.mm

```diff
@@ -32,6 +32,12 @@
 #include <AppKit/AppKit.h>
 #include <ostream>
 
+namespace
+{
+static const std::uint8_t unknownVirtualCode = 0xff;
+static const bool         isIsoKeyboard      = (KBGetLayoutType(LMGetKbdType()) == kKeyboardISO);
```

Comment:
We can remove `static` because unnamed namespaces imply internal linkage since C++11.